components/content_settings/renderer/brave_content_settings_agent_impl.h

```diff
@@ -64,7 +64,8 @@ class BraveContentSettingsAgentImpl
   void BraveSpecificDidAllowJavaScriptOnce(const GURL& details);
   bool AllowAutoplay(bool play_requested) override;
 
-  BraveFarblingLevel GetBraveFarblingLevel() override;
+  BraveFarblingLevel GetBraveFarblingLevel(
+      ContentSettingsType webcompat_settings_type) override;
```

Comment:
maybe we'd like to keep two versions of this method (this one and without the parameter - for all of farbling)?

Reply:
I'm inclined to keep a single method with the parameter, so that we make sure all farbling has a webcompat exception.